Tests/Utilities.Tests.ps1

```diff
@@ -30,20 +30,30 @@ Describe 'Validate various support functions for testing' {
             Test-Path -Path (Join-Path $pathInTemp ".git") -PathType Container | Should -Be $true
             Test-Path -Path (Join-Path $pathInTemp ".git/config") -PathType Leaf | Should -Be $true
 
-            Remove-Item $pathInTemp -Recurse -Force | Out-Null
+            Remove-Item $pathInTemp -Recurse -Force
```

Comment:
Out of curiosity, why did you remove the Out-Null?

Reply:
Not sure. Since it does not show any output in non-debug modes. I removed it to see if it makes any difference. I cross checked with other tests and they have `Out-Null`. So only this file is affected. Just to spark conversations. 😄